package its;

import "CamParameters.proto";

message CoopAwareness {
	required int64 genDeltaTime = 1;
	required its.CamParameters camParameters = 2;
}